### DISCUSSION TITLE
remove character from README.md

### DISCUSSION THREAD

**Discussion by everyonelovespepsicola** [community/community#34181] [Other Feature Feedback, Questions, & Ideas]
i am showing how to make all sh files executable in a forlder, i type this line in

find . -name "*.sh" -exec chmod +x {} \backslash;

but for some reason the "\backslash" at the end is removed when published

**Comment by fadyat** [comment]
Hi, I think you just accidentally left out `backslash` word .

---

Check PR: https://github.com/everyonelovespepsicola/update-AgentDVR-linux/pull/1
